File: Assets/Scripts/Pocket.js
var pocketScore : ParticleEmitter;
var points : int = 10;
var scoreSound : AudioClip;

static var startPoints : int = 0;

function OnTriggerEnter (hit : Collider)
{
	if (hit.gameObject.tag == "OtherBalls")
	{
		Instantiate(pocketScore, transform.position, transform.rotation);
		startPoints += points;
		audio.PlayOneShot(scoreSound);
		GameMaster.canCount = true;
	}
}

File: Assets/Scripts/Other/Pocket.cs
using UnityEngine;
using System.Collections;

public class Pocket : MonoBehaviour
{
    public int points = 10;

    public GameObject pocketEmit;

    public AudioClip scoreSound;
	
	private GameObject gameMaster;
	
    void Awake()
    {
       gameMaster = GameObject.Find("__Game Master");
    }

    void OnTriggerEnter(Collider hit)
    {
        if (hit.transform.tag == "OtherBalls")
        {
			GameMaster.instance.ComboSystem();
			
            //goofy fireworks
            Instantiate(pocketEmit, transform.position, transform.rotation);

            //add the points
            GameMaster gm = gameMaster.GetComponent<GameMaster>();
            gm.AdjustPoints(points);

            audio.PlayOneShot(scoreSound);
			
			//reset the amount of hits
			GameMaster.instance.curHitCount = 0;
			
            //take a ball away from the ball count
            GameMaster.instance.ballCountOnLevel--;
        }
    }

    //void ComboSystem()
    //{
    //    if (GameMaster.instance.ballHitCountScore <= 1)
    //    {
    //        comboCounter++;
    //    }
    //}
}
